Sources/RudderStackAnalytics/Policies/BackoffPolicy/BackoffPolicy.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  BackOffPolicy.swift
+//  RudderStackAnalytics
+//
+//  Created by Satheesh Kannan on 29/08/25.
+//
+
+import Foundation
+
+// MARK: - BackoffPolicy
+/**
+ Protocol representing a backoff policy for retrying operations.
+ Implementations should provide a strategy for calculating the next delay
+ and resetting the backoff state.
+ */
+protocol BackoffPolicy {
+    /**
+     Calculates the next delay in seconds based on the backoff policy.
+     
+     - Returns: The next delay in seconds.
+     */
+    func nextDelayInSeconds() -> Int
```

Comment:
It should be in `millis` to align with the Kotlin interface, refer [BackOffPolicy - Kotlin interface](https://github.com/rudderlabs/rudder-sdk-kotlin/blob/3c521e175b88cd3f20b20362a734a1d010d4bfc5/core/src/main/kotlin/com/rudderstack/sdk/kotlin/core/internals/policies/backoff/BackOffPolicy.kt#L15).
```suggestion
    /**
     Calculates the next delay in milliseconds based on the backoff policy.
     
     - Returns: The next delay in milliseconds.
     */
    func nextDelayInMillis() -> Int
```

CC: @vgupta98

Reply:
We are keeping the delay as well the jitter which is added to it in milliseconds.